islandora_pdf.install

```diff
@@ -46,3 +47,19 @@ function islandora_pdf_update_7001(&$sandbox) {
   $t = get_t();
   return $t('Set colorspace configuration to RBG to maintain existing profile.');
 }
+
+/**
+ * Set and maintain new dUseCIEColor switch variable.
+ */
+function islandora_pdf_update_7100(&$sandbox) {
```

Comment:
I think this should be numbered 7002 if we're following our usual conventions for numbering update hooks